From ac084ae47a87db26fa285ec7990787fc2b62712a Mon Sep 17 00:00:00 2001
From: Dark Dragon <darkdragon-001@web.de>
Date: Sun, 24 Apr 2022 16:51:58 +0200
Subject: [PATCH] Fix metadata targets

Make sure each file is only generated by a single rule.
---
 cpp/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cpp/CMakeLists.txt b/cpp/CMakeLists.txt
index 57261a63..d584461d 100644
--- a/cpp/CMakeLists.txt
+++ b/cpp/CMakeLists.txt
@@ -356,7 +356,7 @@ add_metadata_gen_target (
   ${TEST_METADATA_TARGET}
   "${RESOURCES_DIR}/PhoneNumberMetadataForTesting.xml"
   "test_metadata"
-  "metadata"
+  "test_metadata"
 )
 list (APPEND TESTING_LIBRARY_SOURCES "src/phonenumbers/test_metadata.cc")
 
-- 
2.34.1

